src/react-server-dom-webpack/cjs/react-server-dom-webpack-server.browser.production.js

```
@@ -1910,16 +1910,17 @@ function abort(request, reason) {
       });
       abortListeners.clear();
     }
     null !== request.destination &&
       flushCompletedChunks(request, request.destination);
-  } catch (error$24) {
-    logRecoverableError(request, error$24, null), fatalError(request, error$24);
+  } catch (error$27) {
+    logRecoverableError(request, error$27, null), fatalError(request, error$27);
   }
 }
+var hasOwnProperty = Object.prototype.hasOwnProperty;
```

Biome (2.1.2)

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)